saas_demo/tests/test_saas_demo.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Denis Mudarisov <https://it-projects.info/team/trojikman>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged('post_install', 'at_install')
+class TestSaasDemo(TransactionCase):
+    def setUp(self):
+        super(TestSaasDemo, self).setUp()
+        self.env = self.env(context=dict(
+            self.env.context,
+            test_queue_job_no_delay=True,
+        ))
+        self.saas_demo = self.env['saas.demo'].create({
+            'name': 'SaaS Demo Test'
+        })
+
+        self.saas_operator = self.env['saas.operator'].create({
+            'type': 'local',
+            'demo_id': self.saas_demo.id,
+            'master_url': 'http://saas_demo.127.0.0.1.nip.io:8069',
+        })
+
+        self.saas_module = self.env['saas.module'].create({
+            'name': 'web_login_background',
+        })
```

Comment:
create via demo xml

Reply:
in fact, I realized that we do not need to create this record, since it should be created automatically